README.md

```diff
@@ -31,6 +31,8 @@ Use this extension to quickly create and manage Azure Static Web Apps directly f
 1. Provide the build artifact folder name
     > The path of your build output relative to your apps location. For example, setting a value of 'build' when your app location is set to '/app' will cause the content at '/app/build' to be served.
 
+✨ [See recommended settings for popular frameworks and libraries](https://docs.microsoft.com/en-us/azure/static-web-apps/front-end-frameworks)
```

Comment:
We should register as a aka.ms link.  It'll give us flexibility to change this url without releasing

Reply:
I mean we usually only do that for in-product links. None of the other README links are aka.ms as far as I can tell

Reply:
Always remove the "en-us" part of a url otherwise you're hard-coding english for everyone